dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -6706,10 +6706,16 @@ components:
           format: int64
         organizationName:
           type: string
+        toolsLength:
+          type: integer
```

Comment:
Missing description (line 6716 too) -- are missing the openapi annotation for this in the Java code.